platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegionError.java

```diff
@@ -0,0 +1,47 @@
+package com.mapbox.mapboxsdk.offline;
+
+/**
+ * Created by antonio on 2/21/16.
+ */
+public class OfflineRegionError {
+
+    /**
+     * Success = 1
```

Comment:
I don't think we want to make these specific integer values (which ultimately come from [here](https://github.com/mapbox/mapbox-gl-native/blob/c91b44b02f658548bdc70a625eb54d0ea86f6737/include/mbgl/storage/response.hpp#L41-L45)) part of the public API. What would be an idiomatic way to expose these error types on Android?